src/schema/notifications.ts

```diff
@@ -180,6 +249,11 @@ export class ZimbraNotifications {
 		}
 	};
 
+	private handleAppointmentNotifications = (notification: Notification) => {
+		const items = itemsForKey(notification, 'appt');
```

Comment:
Do we need this variable?

Reply:
Better to have for readability. Also, It's been used in most of the such calls so for consistency as well.